src/test/java/org/jboss/tools/intellij/openshift/validation/DevfileSchemasTest.java

```diff
@@ -11,18 +11,14 @@
 package org.jboss.tools.intellij.openshift.validation;
 
 import com.intellij.testFramework.fixtures.BasePlatformTestCase;
-import com.redhat.devtools.intellij.common.utils.VfsRootAccessHelper;
 import org.jetbrains.yaml.schema.YamlJsonSchemaHighlightingInspection;
 
-import java.io.File;
 
 public class DevfileSchemasTest extends BasePlatformTestCase {
 
     public void testQuarkusDevfile() {
         myFixture.setTestDataPath("src/test/resources");
         myFixture.enableInspections(YamlJsonSchemaHighlightingInspection.class);
-        String path = new File("src").getAbsoluteFile().getParentFile().getAbsolutePath();
```

Comment:
why was this called in the test? It seems to have had no function if it's possible to simply remove it?

Reply:
i really don't know !